datadog/resource_datadog_logs_custom_pipeline_test.go

```diff
@@ -97,6 +98,16 @@ resource "datadog_logs_custom_pipeline" "my_pipeline_test" {
 			sources = ["ip1"]
 		}
 	}
+	processor {
+		lookup_processor {
+			name = "lookup processor"
```

Comment:
we should start only with required fields

Reply:
What's the reasoning? Because all the processors around in the test are defined that way.

Reply:
we should make sure that optional fields are really optional. you are right that there is no example for this case.

Reply:
Still this is a good point, we should create such test. This makes us realize that the same kind of issue may be elsewhere.

Reply:
I have added a test with only the mandatory fields.